mlflow/gateway/runner.py

```diff
@@ -92,14 +95,71 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.stop()
 
 
-def run_app(config_path: str, host: str, port: int, workers: int) -> None:
+class UvicornRunner:
```

Comment:
I started wondering if we really need this.

Reply:
In `test_app.py`, we can mock, we can test dynamic routes there. In `test_runner`, we can just test the reloading logic.